packages/@react-aria/dnd/src/useDrag.ts

```diff
@@ -195,12 +210,67 @@ export function useDrag(options: DragOptions): DragResult {
   };
 
   let modality = useDragModality();
-  let descriptionProps = useDescription(
-    stringFormatter.format(!isDragging ? MESSAGES[modality].start : MESSAGES[modality].end)
-  );
+  let message: string;
+  if (!isDragging) {
+    if (modality === 'touch' && !hasDragButton) {
+      message = 'dragDescriptionLongPress';
+    } else {
+      message = MESSAGES[modality].start;
+    }
+  } else {
+    message = MESSAGES[modality].end;
+  }
+
+  let descriptionProps = useDescription(stringFormatter.format(message));
+
+  let interactions: HTMLAttributes<HTMLElement>;
+  if (!hasDragButton) {
+    // If there's no separate button to trigger accessible drag and drop mode,
+    // then add event handlers to the draggable element itself to start dragging.
+    // For keyboard, we use the Enter key in a capturing listener to prevent other
+    // events such as selection from also occurring. We attempt to infer whether a
+    // pointer event (e.g. long press) came from a touch screen reader, and then initiate
+    // dragging in the native onDragStart listener above.
+
+    interactions = {
+      ...descriptionProps,
+      onPointerDown({nativeEvent: e}) {
```

Comment:
By default, unless you pass `hasDragButton` and merge the resulting `dragButtonProps` through `useButton` onto the element. Then it would just be click.